nestkernel/event.h

```diff
@@ -827,13 +843,20 @@ class GapJEvent : public SecondaryEvent
   static void
   set_syn_id( const synindex synid )
```

Comment:
But here is the problem:

The `add_syn_id`-function is called from `*SecondaryEvent`:
`( get_synapse_prototype( old_id ).get_event() )->add_syn_id( new_id );`
so it has to be defined as `virtual` in the `GapJEvent` base class. And it is not possible for functions to be both virtual and static.

So I do not see any way to get this in one joint function.